NewDawn/NewDawn/HttpUtil/HttpUtil.swift

```diff
@@ -89,4 +89,36 @@ class HttpUtil{
             return URL(string: "http://localhost:8000/api/v1" + final_path)!
         }
     }
+    
+    static func sendAction(user_account_from: String, user_account_to: String, action_type: Int, entity_type: Int, entity_id: Int){
```

Comment:
It would be the most ideal if we pass user id as integers (preferably long integers)

Reply:
Ok hashid is ok